src/test/java/organice/testutil/TypicalPersons.java

```diff
@@ -35,15 +47,22 @@
     public static final Doctor DOCTOR_BENSON = new DoctorBuilder().withNric("T5231426Q")
             .withName("Benson Meier").withPhone("98765432").build();
     public static final Patient PATIENT_CARL = new PatientBuilder().withAge("22").withNric("G5642431P")
-            .withName("Carl Kurz").withPhone("95352563").withPriority("high").build();
+            .withName("Carl Kurz").withPhone("95352563").withPriority("high")
+            .withBloodType("A").withTissueType("1,2,3,4,5,6").withOrgan("kidney")
+            .withDoctorInCharge("S1532142A").build();
     public static final Patient PATIENT_DANIEL = new PatientBuilder().withAge("34").withNric("F6423467F")
-            .withName("Daniel Meier").withPhone("87652533").withPriority("low").build();
+            .withName("Daniel Meier").withPhone("87652533").withPriority("low")
+            .withBloodType("O+").withTissueType("10,5,3,1,6,8").withOrgan("kidney")
+            .withDoctorInCharge("T5231426Q").build();
     public static final Donor DONOR_ELLE = new DonorBuilder().withAge("13").withNric("S9374923S")
-            .withName("Elle Meyer").withPhone("9482224").build();
+            .withName("Elle Meyer").withPhone("9482224").withBloodType("O+")
+            .withTissueType("10,5,3,1,6,8").withOrgan("kidney").withOrganExpiryDate("10-May-2020").build();
     public static final Donor DONOR_FIONA = new DonorBuilder().withAge("25").withNric("F9183156L")
-            .withName("Fiona Kunz").withPhone("9482427").build();
+            .withName("Fiona Kunz").withPhone("9482427").withBloodType("AB+")
+            .withTissueType("7,2,3,9,6,8").withOrgan("kidney").withOrganExpiryDate("10-Jun-2020").build();
     public static final Donor DONOR_GEORGE = new DonorBuilder().withAge("44").withNric("S1234567A")
```

Comment:
Could you do me this favour and reorder the constant declarations, group by person type. i.e. (all the Doctor together and at the top, followed by all the Donor, etc). Unless the present ordering has some significance behind it. Otherwise, could you create an issue, tag it in a follow up comment and resolve this conversation.

Reply:
The on top one is for testing i believe `typicalPerson.json`, the below one more for typical testing like `JsonAdaptedPersonTest`

Reply:
like usually we use `VALID_NAME_BOB` that kind of thing thats the purpose of the below 4

Reply:
Aiight pls make issue thanks